### DISCUSSION TITLE
How can I delete my account?

### DISCUSSION THREAD

**Discussion by ghost** [Pirate-Weather/pirateweather#182] [Q&A] [answered]
I've searched through the web interface, but I can't seem to find a place to delete my account.  I think it's an amazing API, but I simply don't use it anymore, how can I delete my account?

**Comment by cloneofghosts** [comment, marked as answer]
I'll tag @alexander0042 to see if he can look into this for you or you can try emailing api@alexanderrey.ca

**Comment by cloneofghosts** [comment]
Did you ever get this sorted out? If not I can create an issue from this discussion and you may get answers faster that way.

**Reply by ghost** [reply]
I sent the email, but nothing has come out of it yet.  So maybe an issue would be better.  

Thanks for your help so far though!

**Reply by cloneofghosts** [reply]
Created issue #192 for you and will close this discussion.
